docs/getting-started/server/database/_category_.yml

```diff
@@ -1,2 +1,2 @@
 label: "Databases"
-position: 2
+position: 3
```

Comment:
⚠️ As I read through all this I started to feel like the guide in this PR is a sub-page of the server's getting started page since there's repeated content. I think this needs some organizing and you wouldn't need this move.

Reply:
I agree, maintaining the cloud and self-hosted configs are already confusing enough and it would be nice to have a single guide that gives you two options towards the end.

Reply:
I'd love to integrate the container setup into the standard setup guide, but I am not sure how to do so, as even the sections that are shared between the two setups have a few steps that vary slightly.

Is there some way to indicate variance of the setup process _at the step level_ without adding an entire "Dev Container - Bitwarden Dev" "Dev Container - Community Dev" option in the dropdown menu?

Reply:
Maybe [tabs](https://docusaurus.io/docs/markdown-features/tabs)?